clients/src/main/java/org/apache/kafka/common/network/Selector.java

```diff
@@ -529,6 +564,13 @@ public KafkaChannel channel(String id) {
     }
 
     /**
+     * Return the channel with the specified id if it was closed during the last poll().
+     */
+    public KafkaChannel closedChannel(String id) {
```

Comment:
There are currently no references to `KafkaChannel` in `Selectable` and `channel()` is not included in the interface. So I haven't added this.